terraform/main.tf

```diff
@@ -0,0 +1,61 @@
+terraform {
+  required_version = "~> 0.13.0"
+
+  required_providers {
+    newrelic = {
+      source  = "newrelic/newrelic"
+      version = "~> 2.12"
```

Comment:
```suggestion
      version = "~> 2.21"
```

Reply:
bumped this to the most current TF version so we can leverage dashboard apis... 🥳 https://github.com/newrelic/terraform-provider-newrelic/releases/tag/v2.21.0